pkg/apk/impl/fs/rwosfs.go

```diff
@@ -499,18 +499,22 @@ func (f *dirFS) caseSensitiveOnDisk(p string) bool {
 }
 
 // createOnDisk given a path p, determine if it should be created on disk, and, if relevant,
-// add it to the caseMap.
+// add it to the caseMap. If the file already exists on disk, also returns true.
+// This func is responsible solely for determining if you _should_ created it on disk.
+// If that would cause a conflict, that is up to the calling routing to figure out.
 func (f *dirFS) createOnDisk(p string) (createOnDisk bool) {
```

Comment:
can we mod the signature to `func (f *dirFS) createOnDisk(p string) (bool)` if `createOnDisk`  var is never used?

Reply:
Yeah, good point, that was an artifact. Updated.